dangerfile.ts

```diff
@@ -0,0 +1,20 @@
+import { danger, warn } from "danger";
+
+const changeset = danger.git.fileMatch(".changeset/*.md");
+const packageFiles = danger.git.fileMatch(
+  "packages/**/*.tsx",
+  "packages/**/*.tsx"
+);
+const packageFileTests = danger.git.fileMatch(
+  "packages/**/*.test.ts",
+  "packages/**/*.test.tsx"
+);
+
+if (
+  (packageFiles.modified || packageFileTests.modified) &&
+  !changeset.created
+) {
+  fail("You don't have a changeset file for your modifications.");
+} else if (!changeset.created) {
+  warn("You don't have a changeset file for your modifications.");
+}
```

Comment:
Do we need this warning message here?

Reply:
Added this line, just in case, if the files that were changed do not get into the first condition

Reply:
Yeah, I agree with Rich in this case, Alinur. Maybe we don't need to warn when the PR doesn't have changesets. Besides that, now I'm wondering why we have this alert from danger if we already have this bot https://github.com/Localitos/pluto/pull/39#issuecomment-1257852368 🤔

Reply:
The bot doesn't block a PR from being merged without a changeset, since some PRs won't need a changeset. Like overall repo infrastructure work.

Danger is in place to catch a human of merging PRs that need a changeset.

Right now, that warning will happen on every PR that doesn't include a changeset, which is a bit overkill. We only want an error if a package is new or edited, and is missing the changeset file.

Let me know if you have any questions.

Reply:
So, we can remove this bot, right? 

![image](https://user-images.githubusercontent.com/5320963/192570820-ad0d028d-d8a6-4dfa-bbc1-2dcd981d393c.png)

Reply:
We need to keep the bot. We can just remove the warning on the lines in the original comment.